### ISSUE TITLE
episodes don't start at all

### ISSUE THREAD

**Issue by kaumidere** [pystardust/ani-cli#212]
i managed to get it to work yesterday but now whenever i try watching the next episode of golden time the episode simply doesn't start (i have rapid wifi upload and download)

**Comment by Komefumi** [comment]
Coincidence, I have the same problem and I'm trying to watch Golden Time. It was working many hours ago (practically yesterday)

**Comment by hillyu** [comment]
same here, changing base url won't help

**Comment by Vorpalwolf33** [comment]
I have the same issue as well

**Comment by Ananay-g** [comment]
lol for me, it can't search for any anime. shows error.

**Comment by wh1ter4bb1t-js** [comment]
possible duplicate of #202, #207 and #192. can any of you provide a debug log, to confirm this suspicion?

**Comment by wh1ter4bb1t-js** [comment]
@Ananay-g have you tried changing the base URL to be www2.gogoganime.cm or using the solution provided in the current #209 PR?

**Comment by Komefumi** [comment]
> possible duplicate of #202, #207 and #192. can any of you provide a debug log, to confirm this suspicion?

Alight, mine is [here](https://gist.github.com/Komefumi/2079086ab01e74c52abab580cbd7bac2)

I was confused until I finally found #106 and found the `--log-file=debug.log`  has to be added on the same line as `--http-header-fields`, just before the `--http-header-fields` flag

**Comment by wh1ter4bb1t-js** [comment]
yeah, it looks like the same issue as outlined here https://github.com/pystardust/ani-cli/issues/202#issuecomment-999713443. Unfortunately, I don't have a good solution to fix this issue. But it does seem to be a growing issue that needs to be addressed

**Comment by Komefumi** [comment]
> yeah, it looks like the same issue as outlined here [#202 (comment)](https://github.com/pystardust/ani-cli/issues/202#issuecomment-999713443). Unfortunately, I don't have a good solution to fix this issue. But it does seem to be a growing issue that needs to be addressed

I understand, for now we'll wait on this one

**Comment by hillyu** [comment]
parsing ``embedded_video_url`` is fine,  while parsing `video_url` returns nothing, if this can help
Think this sed regex is not working anymore 
```bash
    sed -n -E '
			/^[[:space:]]*sources:/{
			s/.*(https[^'\'']*).*/\1/p
			q
		}
	')
```

**Comment by ura43** [comment]
Closing in favor of #202

**Comment by kaumidere** [comment]
Has this been solved?

**Comment by Ananay-g** [comment]
> @Ananay-g have you tried changing the base URL to be www2.gogoganime.cm or using the solution provided in the current #209 PR?
yes i tried, it didn't work!
